Themes/default/languages/EmailTemplates.english.php

```diff
@@ -356,7 +354,7 @@
 
 {REASON}
 
-You can approve or reject this application by clicking the link below:
+You can approve or reject this application on the link below:
```

Comment:
and here

Reply:
on should be with

Reply:
`from`?

Reply:
You can approve or reject this application on the link below:

Should be

You can approve or reject this application with the link below:

Reply:
Or "at" perhaps?

Reply:
i think we are making it more complicated trying to change it. i think we should just leave it as is.

Reply:
I don't see how any of the options are complicated, but neither do I want to spend time discussing about such a small thing. So if you could just make a decision we'll roll with it.

Reply:
@illori's "using" is the best suggestion.